### DISCUSSION TITLE
2008/06/19/%E5%AE%B6%E9%87%8C%E7%BB%88%E4%BA%8E%E8%BF%9E%E4%B8%8A%E7%BD%91%E4%BA%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#224] [Announcements]
# 2008/06/19/%E5%AE%B6%E9%87%8C%E7%BB%88%E4%BA%8E%E8%BF%9E%E4%B8%8A%E7%BD%91%E4%BA%86

今天请假在家，就是为了装电话和宽带。张江这个地方还是偏僻了点，上网没什选择，就只有电信的ADSL。

https://blog.qizhen.xyz/2008/06/19/%E5%AE%B6%E9%87%8C%E7%BB%88%E4%BA%8E%E8%BF%9E%E4%B8%8A%E7%BD%91%E4%BA%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

秋
2008年06月23日 09:24 编辑
那就得勤劳一点，天天来更新哦~~~~~

回复
WL
2008年06月20日 16:29 编辑
恭喜啊

回复
Wu
2008年06月19日 22:44 编辑
ADSL足够好了，能用就行了。

回复
xiaoji
2008年06月19日 22:39 编辑
快快做功课噢～

回复
yoyo
2008年06月19日 21:30 编辑
好事一桩阿

回复
arrowpig
2008年06月19日 21:18 编辑
可以出书了

回复
